src/Components/Endpoints/src/DependencyInjection/WebAssemblyComponentsEndpointOptions.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Components.Endpoints;
+
+/// <summary>
+/// Options to configure interactive WebAssembly components in a razor components application.
```

Comment:
```suggestion
/// Options to configure interactive WebAssembly components.
```

Just a nit but the terminology gets difficult otherwise. We have so many different names (components, Razor components, Blazor) before getting into the differences between the older and newer hosting styles.

I think we're pushing towards not differentiating types of ASP.NET Core applications and so don't have to caveat what type of app the developer needs to be working with.